city_scrapers/spiders/chi_northern_il_university.py

```diff
@@ -0,0 +1,129 @@
+import re
+from collections import defaultdict
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiNorthernIlUniversitySpider(CityScrapersSpider):
+    name = "chi_northern_il_university"
+    agency = "Northeastern Illinois University"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.neiu.edu/about/board-of-trustees/board-meeting-materials"]
+
+    def parse(self, response):
+        self.link_date_map = defaultdict(list)
+        link_list = response.xpath(
```

Comment:
We don't want to be overly specific in how we search for content in case they change one of the classes, so using CSS selectors instead here will likely be simpler.

Also, since you aren't using `link_list` outside of the loop it might make more sense to just iterate over it directly:

```python
for title in response.css(".field--name-field-generic-body h3"):
```